packages/server/src/service/gm.ts

```diff
@@ -19,7 +19,32 @@ const findByType = (
            : components.find(c => c.types.includes(type))?.long_name)
 }
 
-const rawGeocode = async (query: string): Promise<google.maps.GeocoderResult | null> => {
+const rawGeocode = async (addr: AddressInputParts | string): Promise<google.maps.GeocoderResult | null> => {
+  let query: string
```

Comment:
Instead of using `let`, let's pull this out into a non-exported function `geocodeQuery`